store/table_index.go

```diff
@@ -66,11 +72,18 @@ func generateIndexActions(oldIndexList []*api.Index, indexList []db.Index, datab
 		}
 	}
 	for _, newValue := range indexCreateList {
-		k := newValue.Name
+		k := indexKey{newValue.Name, newValue.Position}
 		if _, ok := oldIndexMap[k]; !ok {
 			creates = append(creates, newValue)
 		}
 	}
+	sort.Slice(deletes, func(i, j int) bool {
+		return deletes[i].ID < deletes[j].ID
+	})
+	sort.Slice(creates, func(i, j int) bool {
+		return creates[i].Name < creates[j].Name || (creates[i].Name == creates[j].Name && creates[i].Position == creates[j].Position)
```

Comment:
Not quite understand the "creates[i].Name == creates[j].Name && creates[i].Position == creates[j].Position" part

Reply:
Done.